.github/workflows/port-pr.yaml

```diff
@@ -0,0 +1,93 @@
+name: Port PR
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  port-pr:
+    runs-on: ubuntu-latest
+    if: (startsWith(github.event.comment.body, '/backport') || startsWith(github.event.comment.body, '/forwardport')) && github.event.issue.pull_request
+    steps:
+      - name: Check org membership
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          if gh api orgs/${GITHUB_REPOSITORY_OWNER}/members --paginate | jq -e --arg GITHUB_ACTOR "$GITHUB_ACTOR" '.[] | select(.login == $GITHUB_ACTOR)' > /dev/null; then
+              echo "${GITHUB_ACTOR} is a member"
+              echo "is_member=true" >> $GITHUB_ENV
+          else
+              echo "${GITHUB_ACTOR} is not a member" >> $GITHUB_STEP_SUMMARY
+              echo "is_member=false" >> $GITHUB_ENV
+          fi
+      - name: Check milestone
+        if: ${{ env.is_member == 'true' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ORIGINAL_ISSUE_NUMBER: ${{ github.event.issue.number }}
+          COMMENT_BODY: ${{ github.event.comment.body }}
+        run: |
+          MILESTONE=$(echo "${COMMENT_BODY}" | awk '{ print $2 }')
```

Comment:
Comment only. I'm most confused by the extraction of MS from comment. 

`.github/workflows/port-issue.yaml`
```
BODY_MILESTONE=$(echo "${COMMENT_BODY}" | awk '{ print $2 }')
MILESTONE=${BODY_MILESTONE//[^a-zA-Z0-9\-\.]/}
```
Will fail if there are line endings at the end of `/backport v2.8.next1` in a gh issue comment. I've added some extra debug there to help that scenario https://github.com/rancher/dashboard/pull/10486/files

This though doesn't sanitised the value at all. We can either update this part to also use the milestone ... and fix in both places when one has been identified OR merge as is and see if there's issues in the future.

Reply:
You could use `tr` to normalize the string to convert any newlines with spaces before defining the `BODY_MILESTONE`, something like:
```bash
NORMALIZED_BODY=$(echo -e "${COMMENT_BODY}" | tr '\n' ' ')
BODY_MILESTONE=$(echo -e "${NORMALIZED_BODY}" | awk '{ print $2 }')
MILESTONE=${BODY_MILESTONE//[^a-zA-Z0-9\-\.]/}
```

Reply:
I'm sure that we'll find additional changes to make after merging and testing, but it's good to identify these things now.

I've updated this portion so that it matches what we currently do in `port-issue.yaml`. These two workflows are largely similar, so I think that it makes sense to keep them aligned.

Reply:
> NORMALIZED_BODY=$(echo -e "${COMMENT_BODY}" | tr '\n' ' ')

I was testing the other PR and it successfully caught `\n` newlines in the job before. So really not sure how it failed with a new line in a comment. Think we can wait and see